### ISSUE TITLE
2025-06-05, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#191]
# Today's Arxiv Papers

(1) [Language-Image Alignment with Fixed Text Encoders](http://arxiv.org/abs/2506.04209v1), 2025-06-04

(2) [Advancing Multimodal Reasoning: From Optimized Cold Start to Staged Reinforcement Learning](http://arxiv.org/abs/2506.04207v1), 2025-06-04

(3) [EPiC: Towards Lossless Speedup for Reasoning Training through Edge-Preserving CoT Condensation](http://arxiv.org/abs/2506.04205v1), 2025-06-04

(4) [Cascadia: A Cascade Serving System for Large Language Models](http://arxiv.org/abs/2506.04203v1), 2025-06-04

(5) [TracLLM: A Generic Framework for Attributing Long Context LLMs](http://arxiv.org/abs/2506.04202v1), 2025-06-04

(6) [R-Search: Empowering LLM Reasoning with Search via Multi-Reward Reinforcement Learning](http://arxiv.org/abs/2506.04185v1), 2025-06-04

(7) [SuperWriter: Reflection-Driven Long-Form Generation with Large Language Models](http://arxiv.org/abs/2506.04180v1), 2025-06-04

(8) [SkipGPT: Dynamic Layer Pruning Reinvented with Token Awareness and Module Decoupling](http://arxiv.org/abs/2506.04179v1), 2025-06-04

(9) [Does Prompt Design Impact Quality of Data Imputation by LLMs?](http://arxiv.org/abs/2506.04172v1), 2025-06-04

(10) [Neural and Cognitive Impacts of AI: The Influence of Task Subjectivity on Human-LLM Collaboration](http://arxiv.org/abs/2506.04167v1), 2025-06-04

(11) [N$^2$: A Unified Python Package and Test Bench for Nearest Neighbor-Based Matrix Completion](http://arxiv.org/abs/2506.04166v1), 2025-06-04

(12) [VISCA: Inferring Component Abstractions for Automated End-to-End Testing](http://arxiv.org/abs/2506.04161v1), 2025-06-04

(13) [A Dataset for Addressing Patient's Information Needs related to Clinical Course of Hospitalization](http://arxiv.org/abs/2506.04156v1), 2025-06-04

(14) [Establishing Trustworthy LLM Evaluation via Shortcut Neuron Analysis](http://arxiv.org/abs/2506.04142v1), 2025-06-04

(15) [MMR-V: What's Left Unsaid? A Benchmark for Multimodal Deep Reasoning in Videos](http://arxiv.org/abs/2506.04141v1), 2025-06-04

(16) [Are Lexicon-Based Tools Still the Gold Standard for Valence Analysis in Low-Resource Flemish?](http://arxiv.org/abs/2506.04139v1), 2025-06-04

(17) [TRiSM for Agentic AI: A Review of Trust, Risk, and Security Management in LLM-based Agentic Multi-Agent Systems](http://arxiv.org/abs/2506.04133v1), 2025-06-04

(18) [Guided Speculative Inference for Efficient Test-Time Alignment of LLMs](http://arxiv.org/abs/2506.04118v1), 2025-06-04

(19) [Rectified Sparse Attention](http://arxiv.org/abs/2506.04108v1), 2025-06-04

(20) [TextAtari: 100K Frames Game Playing with Language Agents](http://arxiv.org/abs/2506.04098v1), 2025-06-04

(21) [AmbiK: Dataset of Ambiguous Tasks in Kitchen Environment](http://arxiv.org/abs/2506.04089v1), 2025-06-04

(22) [Multimodal Tabular Reasoning with Privileged Structured Information](http://arxiv.org/abs/2506.04088v1), 2025-06-04

(23) [EuroLLM-9B: Technical Report](http://arxiv.org/abs/2506.04079v1), 2025-06-04

(24) [LLMEval-Med: A Real-world Clinical Benchmark for Medical LLMs with Physician Validation](http://arxiv.org/abs/2506.04078v1), 2025-06-04

(25) [A Novel Data Augmentation Approach for Automatic Speaking Assessment on Opinion Expressions](http://arxiv.org/abs/2506.04077v1), 2025-06-04

(26) [Controlling Difficulty of Generated Text for AI-Assisted Language Learning](http://arxiv.org/abs/2506.04072v1), 2025-06-04

(27) [LaF-GRPO: In-Situ Navigation Instruction Generation for the Visually Impaired via GRPO with LLM-as-Follower Reward](http://arxiv.org/abs/2506.04070v1), 2025-06-04

(28) [Progressive Mastery: Customized Curriculum Learning with Guided Prompting for Mathematical Reasoning](http://arxiv.org/abs/2506.04065v1), 2025-06-04

(29) [Crowd-SFT: Crowdsourcing for LLM Alignment](http://arxiv.org/abs/2506.04063v1), 2025-06-04

(30) [High Accuracy, Less Talk (HALT): Reliable LLMs through Capability-Aligned Finetuning](http://arxiv.org/abs/2506.04051v1), 2025-06-04

(31) [Explainability-Based Token Replacement on LLM-Generated Text](http://arxiv.org/abs/2506.04050v1), 2025-06-04

(32) [Lacuna Inc. at SemEval-2025 Task 4: LoRA-Enhanced Influence-Based Unlearning for LLMs](http://arxiv.org/abs/2506.04044v1), 2025-06-04

(33) [Think Like a Person Before Responding: A Multi-Faceted Evaluation of Persona-Guided LLMs for Countering Hate](http://arxiv.org/abs/2506.04043v1), 2025-06-04

(34) [Unveiling and Eliminating the Shortcut Learning for Locate-Then-Edit Knowledge Editing via Both Subject and Relation Awareness](http://arxiv.org/abs/2506.04042v1), 2025-06-04

(35) [LexTime: A Benchmark for Temporal Ordering of Legal Events](http://arxiv.org/abs/2506.04041v1), 2025-06-04

(36) [Mitigating Hallucinations in Large Vision-Language Models via Entity-Centric Multimodal Preference Optimization](http://arxiv.org/abs/2506.04039v1), 2025-06-04

(37) [Generating Automotive Code: Large Language Models for Software Development and Verification in Safety-Critical Systems](http://arxiv.org/abs/2506.04038v1), 2025-06-04

(38) [Privacy and Security Threat for OpenAI GPTs](http://arxiv.org/abs/2506.04036v1), 2025-06-04

(39) [AI Agents for Conversational Patient Triage: Preliminary Simulation-Based Evaluation with Real-World EHR Data](http://arxiv.org/abs/2506.04032v1), 2025-06-04

(40) [CETBench: A Novel Dataset constructed via Transformations over Programs for Benchmarking LLMs for Code-Equivalence Checking](http://arxiv.org/abs/2506.04019v1), 2025-06-04

(41) [AgentMisalignment: Measuring the Propensity for Misaligned Behaviour in LLM-Based Agents](http://arxiv.org/abs/2506.04018v1), 2025-06-04

(42) [GORACS: Group-level Optimal Transport-guided Coreset Selection for LLM-based Recommender Systems](http://dx.doi.org/10.1145/3711896.3736985), 2025-06-04

(43) [DynTok: Dynamic Compression of Visual Tokens for Efficient and Effective Video Understanding](http://arxiv.org/abs/2506.03990v1), 2025-06-04

(44) [Around the World in 24 Hours: Probing LLM Knowledge of Time and Place](http://arxiv.org/abs/2506.03984v1), 2025-06-04

(45) [From Real to Synthetic: Synthesizing Millions of Diversified and Complicated User Instructions with Attributed Grounding](http://arxiv.org/abs/2506.03968v1), 2025-06-04

(46) [TableEval: A Real-World Benchmark for Complex, Multilingual, and Multi-Structured Table Question Answering](http://arxiv.org/abs/2506.03949v1), 2025-06-04

(47) [Automatic Multi-level Feature Tree Construction for Domain-Specific Reusable Artifacts Management](http://arxiv.org/abs/2506.03946v1), 2025-06-04

(48) [Graph Counselor: Adaptive Graph Exploration via Multi-Agent Synergy to Enhance LLM Reasoning](http://arxiv.org/abs/2506.03939v1), 2025-06-04

(49) [VisCoder: Fine-Tuning LLMs for Executable Python Visualization Code Generation](http://arxiv.org/abs/2506.03930v1), 2025-06-04

(50) [Vision Remember: Alleviating Visual Forgetting in Efficient MLLM with Vision Feature Resample](http://arxiv.org/abs/2506.03928v1), 2025-06-04

(51) [More or Less Wrong: A Benchmark for Directional Bias in LLM Comparative Reasoning](http://arxiv.org/abs/2506.03923v1), 2025-06-04

(52) [HSSBench: Benchmarking Humanities and Social Sciences Ability for Multimodal Large Language Models](http://arxiv.org/abs/2506.03922v1), 2025-06-04

(53) [Boosting Open-Source LLMs for Program Repair via Reasoning Transfer and LLM-Guided Reinforcement Learning](http://arxiv.org/abs/2506.03921v1), 2025-06-04

(54) [Magic Mushroom: A Customizable Benchmark for Fine-grained Analysis of Retrieval Noise Erosion in RAG Systems](http://arxiv.org/abs/2506.03901v1), 2025-06-04

(55) [Pre$^3$: Enabling Deterministic Pushdown Automata for Faster Structured LLM Generation](http://arxiv.org/abs/2506.03887v1), 2025-06-04

(56) [RadialRouter: Structured Representation for Efficient and Robust Large Language Models Routing](http://arxiv.org/abs/2506.03880v1), 2025-06-04

(57) [Evaluating Apple Intelligence's Writing Tools for Privacy Against Large Language Model-Based Inference Attacks: Insights from Early Datasets](http://arxiv.org/abs/2506.03870v1), 2025-06-04

(58) [EuroGEST: Investigating gender stereotypes in multilingual language models](http://arxiv.org/abs/2506.03867v1), 2025-06-04

(59) [PulseReddit: A Novel Reddit Dataset for Benchmarking MAS in High-Frequency Cryptocurrency Trading](http://arxiv.org/abs/2506.03861v1), 2025-06-04

(60) [Prompt Candidates, then Distill: A Teacher-Student Framework for LLM-driven Data Annotation](http://arxiv.org/abs/2506.03857v1), 2025-06-04

(61) [Vulnerability-Aware Alignment: Mitigating Uneven Forgetting in Harmful Fine-Tuning](http://arxiv.org/abs/2506.03850v1), 2025-06-04

(62) [AssetOpsBench: Benchmarking AI Agents for Task Automation in Industrial Asset Operations and Maintenance](http://arxiv.org/abs/2506.03828v1), 2025-06-04

(63) [From Theory to Practice: Real-World Use Cases on Trustworthy LLM-Driven Process Modeling, Prediction and Automation](http://arxiv.org/abs/2506.03801v1), 2025-06-04

(64) [STELLA: Towards Protein Function Prediction with Multimodal LLMs Integrating Sequence-Structure Representations](http://arxiv.org/abs/2506.03800v1), 2025-06-04

(65) [Mark My Words: A Robust Multilingual Model for Punctuation in Text and Speech Transcripts](http://arxiv.org/abs/2506.03793v1), 2025-06-04

(66) [Knockout LLM Assessment: Using Large Language Models for Evaluations through Iterative Pairwise Comparisons](http://arxiv.org/abs/2506.03785v1), 2025-06-04

(67) [Unifying Uniform and Binary-coding Quantization for Accurate Compression of Large Language Models](http://arxiv.org/abs/2506.03781v1), 2025-06-04

(68) [ClozeMath: Improving Mathematical Reasoning in Language Models by Learning to Fill Equations](http://arxiv.org/abs/2506.03763v1), 2025-06-04

(69) [AhaKV: Adaptive Holistic Attention-Driven KV Cache Eviction for Efficient Inference of Large Language Models](http://arxiv.org/abs/2506.03762v1), 2025-06-04

(70) [Act-as-Pet: Benchmarking the Abilities of Large Language Models as E-Pets in Social Network Services](http://arxiv.org/abs/2506.03761v1), 2025-06-04

(71) [Understanding Physical Properties of Unseen Deformable Objects by Leveraging Large Language Models and Robot Actions](http://arxiv.org/abs/2506.03760v1), 2025-06-04

(72) [Spatiotemporal Prediction of Electric Vehicle Charging Load Based on Large Language Models](http://arxiv.org/abs/2506.03728v1), 2025-06-04

(73) [Sign-SGD is the Golden Gate between Multi-Node to Single-Node Learning: Significant Boost via Parameter-Free Optimization](http://arxiv.org/abs/2506.03725v1), 2025-06-04

(74) [Verbalized Confidence Triggers Self-Verification: Emergent Behavior Without Explicit Reasoning Supervision](http://arxiv.org/abs/2506.03723v1), 2025-06-04

(75) [ScoreRAG: A Retrieval-Augmented Generation Framework with Consistency-Relevance Scoring and Structured Summarization for News Generation](http://arxiv.org/abs/2506.03704v1), 2025-06-04

(76) [Learning-at-Criticality in Large Language Models for Quantum Field Theory and Beyond](http://arxiv.org/abs/2506.03703v1), 2025-06-04

(77) [AdaDecode: Accelerating LLM Decoding with Adaptive Layer Parallelism](http://arxiv.org/abs/2506.03700v1), 2025-06-04

(78) [Scaling Transformers for Discriminative Recommendation via Generative Pretraining](http://dx.doi.org/10.1145/3711896.3737117), 2025-06-04

(79) [A Two-Staged LLM-Based Framework for CI/CD Failure Detection and Remediation with Industrial Validation](http://arxiv.org/abs/2506.03691v1), 2025-06-04

(80) [Robust Preference Optimization via Dynamic Target Margins](http://arxiv.org/abs/2506.03690v1), 2025-06-04

(81) [Reason from Future: Reverse Thought Chain Enhances LLM Reasoning](http://arxiv.org/abs/2506.03673v1), 2025-06-04

(82) [Trustworthy Medical Question Answering: An Evaluation-Centric Survey](http://arxiv.org/abs/2506.03659v1), 2025-06-04

(83) [Client-Side Zero-Shot LLM Inference for Comprehensive In-Browser URL Analysis](http://arxiv.org/abs/2506.03656v1), 2025-06-04

(84) [Facts are Harder Than Opinions -- A Multilingual, Comparative Analysis of LLM-Based Fact-Checking Reliability](http://arxiv.org/abs/2506.03655v1), 2025-06-04

(85) [Mono: Is Your "Clean" Vulnerability Dataset Really Solvable? Exposing and Trapping Undecidable Patches and Beyond](http://arxiv.org/abs/2506.03651v1), 2025-06-04

(86) [RewardAnything: Generalizable Principle-Following Reward Models](http://arxiv.org/abs/2506.03637v1), 2025-06-04

(87) [Robustness of Prompting: Enhancing Robustness of Large Language Models Against Prompting Attacks](http://arxiv.org/abs/2506.03627v1), 2025-06-04

(88) [Do Large Language Models Know Folktales? A Case Study of Yokai in Japanese Folktales](http://arxiv.org/abs/2506.03619v1), 2025-06-04

(89) [Learning to Insert [PAUSE] Tokens for Better Reasoning](http://arxiv.org/abs/2506.03616v1), 2025-06-04

(90) [Orak: A Foundational Benchmark for Training and Evaluating LLM Agents on Diverse Video Games](http://arxiv.org/abs/2506.03610v1), 2025-06-04

(91) [Auto prompt sql: a resource-efficient architecture for text-to-sql translation in constrained environments](http://arxiv.org/abs/2506.03598v1), 2025-06-04

(92) [From Understanding to Generation: An Efficient Shortcut for Evaluating Language Models](http://arxiv.org/abs/2506.03592v1), 2025-06-04

(93) [Resolving Task Objective Conflicts in Unified Multimodal Understanding and Generation via Task-Aware Mixture-of-Experts](http://arxiv.org/abs/2506.03591v1), 2025-06-04

(94) [Preface to the Special Issue of the TAL Journal on Scholarly Document Processing](http://arxiv.org/abs/2506.03587v1), 2025-06-04

(95) [Improving LLM-Based Fault Localization with External Memory and Project Context](http://arxiv.org/abs/2506.03585v1), 2025-06-04

(96) [Exchange of Perspective Prompting Enhances Reasoning in Large Language Models](http://arxiv.org/abs/2506.03573v1), 2025-06-04

(97) [FreePRM: Training Process Reward Models Without Ground Truth Process Labels](http://arxiv.org/abs/2506.03570v1), 2025-06-04

(98) [POSS: Position Specialist Generates Better Draft for Speculative Decoding](http://arxiv.org/abs/2506.03566v1), 2025-06-04

(99) [ConsistentChat: Building Skeleton-Guided Consistent Dialogues for Large Language Models from Scratch](http://arxiv.org/abs/2506.03558v1), 2025-06-04

(100) [BPO: Revisiting Preference Modeling in Direct Preference Optimization](http://arxiv.org/abs/2506.03557v1), 2025-06-04
